tree/ntuple/v7/inc/ROOT/RNTupleProcessor.hxx

```diff
@@ -244,6 +249,10 @@ public:
    /// \return A pointer to the newly created RNTupleProcessor.
    static std::unique_ptr<RNTupleProcessor>
    CreateChain(const std::vector<RNTupleSourceSpec> &ntuples, std::unique_ptr<RNTupleModel> model = nullptr);
```

Comment:
Not for this PR, but something I noticed. Let's keep in mind that having public APIs in C++ with `std::unique_ptr` as parameter actively prevents using that API from Python, because of https://github.com/root-project/root/issues/14425